lab_2_retrieval_w_bm25/main.py

```diff
@@ -19,6 +21,22 @@ def tokenize(text: str) -> list[str] | None:
     In case of corrupt input arguments, None is returned.
     """
 
+    if not isinstance(text, str):
+        return None
+
+    tokenized_list = []
+
+    for elem in text:
+        if not elem.isalpha():
+            text.replace(elem, ' ')
+
+    text.lower()
+
+    for elem in text.split():
+        if elem.isalpha():
```

Comment:
is it really need to have two `isalpha` checks?

Reply:
fixed